packages/core/addon/models/bard-request-v2/request.ts

```diff
@@ -76,7 +76,7 @@ export default class RequestFragment extends Fragment.extend(Validations) implem
   columns!: FragmentArray<ColumnFragment>;
 
   @attr('string')
-  table!: string;
+  table!: string; // TODO this can be undefined
```

Comment:
??

Reply:
This came up in the dataSource selector since I allowed a report to be passed in with only the `{ dataSource: 'name' }` property being set

Reply:
```suggestion
  table!: string; // TODO this can be undefined when only `dataSource` is set
```